src/main/drivers/max7456.c

```diff
@@ -757,6 +760,37 @@ void max7456WriteNvm(uint8_t char_address, const uint8_t *font_data)
     max7456Lock = false;
 }
 
+void max7456ReadNvm(uint8_t char_address, uint8_t *font_data)
+{
+#ifdef MAX7456_DMA_CHANNEL_TX
+    while (dmaTransactionInProgress);
+#endif
+    while (max7456Lock);
```

Comment:
If `max7456Lock` is used only in main code (outside of interrupts), it will be always false here.
Otherwise the code will freeze when it is set when testing it.